eagleproject/core/blockchain/harvest/snapshots.py

```diff
@@ -103,20 +105,22 @@ def _build_asset_block_oeth(symbol, block_number):
     # TODO: Rename these columns??
     ora_tok_usd_min = 0
     ora_tok_usd_max = 0
+    if symbol == "ETH":
+        ora_tok_usd_max = 1
+        ora_tok_usd_min = 1
+    else:
+        if ora_tok_usd_min < 0:
```

Comment:
will this ever hit? looks like you are setting `ora_tok_usd_min = 0` just above

Reply:
Thanks for catching this. Yeah, it's my bad, the check isn't necessary here